### ISSUE TITLE
SVG tabindex="-1" is keyboard focusable in Internet Explorer

### ISSUE THREAD

**Issue by rodneyrehm** [medialize/ally.js#37]
`<svg tabindex="-1">` is still keyboard focusable (tabbable) in Internet Explorer and `ally/element/disabled` does not work around that yet

**Comment by rodneyrehm** [comment]
possibly related to #7
